ignite-generator/component/templates/component-test.js.template

```diff
@@ -0,0 +1,24 @@
+import test from 'ava'
+import React from 'react'
+import { shallow } from 'enzyme'
+import <%= name %> from '../../App/Components/<%= name %>'
+
+test('component exists', t => {
+  const wrapper = shallow(<<%= name %>  />)
```

Comment:
aah, yes, got you now. Its clear now that you've explained it. I'll annotate the template to make it clear that this is what is happening.